src/main/java/net/sf/jabref/model/metadata/MetaData.java

```diff
@@ -209,8 +209,15 @@ private void postChange() {
     }
 
     public void setEncoding(Charset encoding) {
+        setEncoding(encoding, true);
+    }
+
+    // The parameter postChanges has been introduced to avoid event loops while saving a database
```

Comment:
Why not change it to a javadoc? Then one can see it, if he wants to call the method ;)